controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
+ */
+module.exports.readAll = async (req, res) => {
+  const allGenres = await Genre.find({});
+
+  if (res) res.json(allGenres);
+  else return allGenres;
+};
+
+module.exports.readOne = async (req, res) => {
+  const foundGenre = await Genre.findById(req.params.id);
+  res.json(foundGenre);
+};
+
+module.exports.readOneByKey = key =>
+  Genre.findOne({ 'external_ids.tmdb_id': key }).exec();
+
```

Comment:
Also, it's cleaner if you have one export thing at the very end, instead of for each method. That also will make each line a little shorter and cleaner, allowing the above format.

(This should probably be a few lines lower, below the create)
```suggestion
module.exports = {
  readAll,
  readOne,
  readOneByKey,
  create,
};
```